Jenkinsfile

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/groovy
+
+String slackChannel = 'dpe-build-notification'
+def decodedJobName = env.JOB_NAME.replaceAll("%2F", "/")
+
+pipeline {
+
+    options {
+        disableConcurrentBuilds()
+        buildDiscarder logRotator(artifactDaysToKeepStr: '5', artifactNumToKeepStr: '5', numToKeepStr: '10')
+        timeout(time: 10, unit: 'MINUTES')
+    }
+
+    agent {
+        kubernetes {
+            yamlFile 'builderPodTemplate.yaml'
+            defaultContainer 'talend-tsbi-dev'
+        }
+    }
+
+    environment {
+        ARTIFACTORY_URL = 'https://artifactory.datapwn.com/artifactory'
+        PUBLIC_REPO_URL = 'https://talend.github.io/helm-charts-public'
+        GIT_USER = 'ci-build-dpe'
+        GIT_EMAIL = 'ci-build-dpe@talend.com'
+        GIT_CREDS = credentials('github-credentials')
+    }
+
+    parameters {
+        string(name: 'CHART_NAME', defaultValue: '', description: 'Chart name + version to the internal chart to deploy (ie: dpe-remote-engine-client-1.2.0)')
+        string(name: 'CHART_REPO', defaultValue: 'tlnd-helm-ce-dev', description: 'The repo to use')
+        string(name: 'CHART_PUBLIC_FOLDER', defaultValue: 'engine', description: 'Chart root folder')
+    }
+
+    stages {
+        stage('Initialisation') {
+            steps {
+                sh '''
+                     #! /bin/bash
+                     set +x
+
+                     echo https://$GIT_CREDS_USR:$GIT_CREDS_PSW@github.com > /tmp/.git-credentials
+                     git config credential.helper 'store --file /tmp/.git-credentials\'
+                     git config user.name "$GIT_USER"
+                     git config user.email "$GIT_EMAIL"
+                  '''
+            }
+        }
+
+        stage('Download chart') {
+            steps {
+                withCredentials([
+                        usernamePassword(credentialsId: 'artifactory-datapwn-credentials', passwordVariable: 'ARTIFACTORY_DOCKER_PASSWORD', usernameVariable: 'ARTIFACTORY_DOCKER_LOGIN')
+                ]) {
+                    sh '''
+                        mkdir $CHART_PUBLIC_FOLDER/$CHART_NAME
+                        cd $CHART_PUBLIC_FOLDER/$CHART_NAME
+                        helm pull $ARTIFACTORY_URL/$CHART_REPO/$CHART_NAME.tgz --username $ARTIFACTORY_DOCKER_LOGIN --password $ARTIFACTORY_DOCKER_PASSWORD
```

Comment:
Don't you want to use the `--version` flag where the user can write the version of the chart to pull ?
Also maybe it would be interesting to use the `--untar=false`

Reply:
i can give a try to the --version
but i don't see the point of the --untar=false as it's the default value

Reply:
it's not working, i think it's not compatible with the full chart url (u have to use a helm repo)